docker2earthly/convert.go

```diff
@@ -64,6 +69,15 @@ func Docker2Earthly(dockerfilePath, earthfilePath, imageTag string) error {
 		targets = append(targets, []string{
 			fmt.Sprintf("FROM %s", stage.BaseName),
 		})
+		// These args are in scope *only* for the very first FROM
+		if i == 0 && len(initialArgs) > 0 {
+			var fromArgs []string
+			for _, arg := range initialArgs {
+				fromArgs = append(fromArgs, fmt.Sprintf("%s", arg.String()))
+			}
+			targets[i+1] = append(fromArgs, targets[i+1]...)
```

Comment:
this assumption (`i+1`) feels brittle; would it be possible to do something more along the lines of:

```
for i, stage := range stages {
    fromCmd := []string{fmt.Sprintf("FROM %s", stage.BaseName)
    // These args are in scope *only* for the very first FROM
    if i == 0 && len(initialArgs) > 0 {
        var fromArgs []string
            for _, arg := range initialArgs {
                fromArgs = append(fromArgs, fmt.Sprintf("%s", arg.String()))
            }
        fromCmd = append(fromArgs, fromCmd...)
    } 
    targets = append(targets, fromCmd)
}
```

Reply:
Absolutely (and I agree).  I had reasons for going that way, but I should have followed my instincts on this one!

I'll fix the lint issue at the same time.  I assumed `+test` would always be sufficient to guard against embarrassing failures like that.  I assume not including `+lint` inside `+test` is intentional; would it worth throwing a quick snippet in the contributors file (to advise explicitly running the linter prior to pushing)?